queries.sql

```diff
@@ -21,3 +21,74 @@ WHERE name != 'Gabumon';
 
 SELECT * FROM animals
 WHERE weight_kg between 10.4 and 17.3;
+
+-- changing the species 
+
+update animals
+set species = 'digimon'
+where name like '%mon';
+
+update animals
+set species = 'pokemon'
+where species is null;
+
+-- deleting animals and rollback
+
+begin;
+
+delete from animals;
+
+rollback;
```

Comment:
- [ ] Please make sure to add a statement to verify if all records in the animals table still exist after rolling back the deleted animals table as specified in the project requirements.